File: 1635-number-of-good-pairs/number-of-good-pairs.c
int numIdenticalPairs(int* nums, int numsSize) {
    int count =0;
    for(int i =0;i<numsSize;i++){
        for(int j=i;j<numsSize;j++){
            if(nums[i]==nums[j]){
                if(i<j){
                    count++;
                }
            }
        }
    }
    return count;
}

File: 1635-number-of-good-pairs/number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        Integer count =0;
        Integer i;
        Integer j;
        for(i=0;i<nums.length;i++){
           for(j=i;j<nums.length;j++){
            if(nums[i]==nums[j]){
                if(i<j){
                    count++;
                }
            }
           }

        }
        return count;
    }

}